r/content/index.Rmd

```diff
@@ -44,5 +44,8 @@ If you have any requests for new recipes, please open a ticket via
 [the cookbook's GitHub Issues page](https://github.com/apache/arrow-cookbook/issues).
 
 If you have any Arrow feature requests to make or bugs to report, please 
-[open an issue on the project JIRA](https://issues.apache.org/jira/projects/ARROW/issues)
+[open an issue on the project GitHub repo](https://github.com/apache/arrow/issues)
 
+## How to cite
+
+Crane, N, Hazlitt, S, et al (2023). Apache Arrow R Cookbook. https://arrow.apache.org/cookbook/r/
```

Comment:
```suggestion
Crane N, Hazlitt S, et al (2023). Apache Arrow R Cookbook. https://arrow.apache.org/cookbook/r/
```

I picked https://dplyr.tidyverse.org/authors.html at a bit random, but that doesn't have the commas between last name and first initial. I'm ok to keep it if we want it, but find the commas a bit confusing there since they separate both full names and first-last names?